packages/server/lib/cloud/studio/StudioLifecycleManager.ts

```diff
@@ -195,6 +291,10 @@ export class StudioLifecycleManager {
     if (this.studioManager) {
       debug('Studio ready - calling listener immediately')
       listener(this.studioManager)
+
+      if (process.env.CYPRESS_LOCAL_STUDIO_PATH) {
```

Comment:
Suggest adding a comment here. I assume we are adding the listener so when we hot reload, we can call the listener.

Reply:
[`7b14bf8` (#31796)](https://github.com/cypress-io/cypress/pull/31796/commits/7b14bf8fe187b0e496f7e5c826a0320b75b7a409)